spec.html

```diff
@@ -31134,15 +31129,12 @@ <h1>GetMatchIndicesArray ( _S_, _match_ )</h1>
 
         <emu-clause id="sec-makeindicesarray" aoid="MakeIndicesArray">
           <h1>MakeIndicesArray ( _S_, _indices_, _groupNames_, _hasGroups_ )</h1>
-          <p>The abstract operation MakeIndicesArray takes arguments _S_, _indices_, _groupNames_, and _hasGroups_. It performs the following steps when called:</p>
+          <p>The abstract operation MakeIndicesArray takes arguments _S_ (a String), _indices_ (a List), _groupNames_ (a List), and _hasGroups_ (a Boolean). It performs the following steps when called:</p>
           <emu-alg>
-            1. Assert: Type(_S_) is String.
-            1. Assert: _indices_ is a List.
             1. Let _n_ be the number of elements in _indices_.
             1. Assert: _n_ &lt; 2<sup>32</sup> - 1.
-            1. Assert: _groupNames_ is a List with _n_ - 1 elements.
+            1. Assert: _groupNames_ with _n_ - 1 elements.
```

Comment:
```suggestion
            1. Assert: _groupNames_ has _n_ - 1 elements.
```
?

Reply:
For whatever reason, github is not allowing me to commit suggestions. I'll push a new commit shortly.

Reply:
whenever the suggester has write access, github doesn't allow you to commit the suggestion. it's a long-standing bug.